src/renderer/pages/project/chat.vue

```
@@ -432,13 +432,23 @@
   return enLang.languages[mulmoScriptHistoryStore.currentMulmoScript.lang ?? "en"];
 });
 
+const scriptLangLocalize = computed(() => {
+  return t("languages." + mulmoScriptHistoryStore.currentMulmoScript.lang ?? "en");
```

ESLint

[error] Unexpected constant nullishness on the left-hand side of a `??` expression.

(no-constant-binary-expression)